common/v2/vendor/react-use/useLocalStorage.ts

```diff
@@ -1,7 +1,6 @@
-import { useEffect, useState } from 'react';
+import { noOp } from 'v2/utils';
```

Comment:
Since this is no longer a hook, lets simplify the code.
We can turn merge it with `useDatabase` and turn it into a DatabaseService.

Reply:
Hook useLocalStorage merged with useDatabase transformed into DatabaseService.